### PR TITLE
Update go version, add tools for verification and testing

### PR DESCRIPTION
**Pull request by coreydaley**
Fixes #

**Summary of Changes**

1.
2. 
3.

> PS: Make sure your PR includes/updates tests! If you need help with this part, just ask!

### CODE REVIEW

**Comment by coreydaley** [comment]
@apoorvajagtap I think this one is ready for review

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 context
```
Comment (apoorvajagtap) [inline README.md:1]:
```suggestion
# gorilla/context
```

Comment (apoorvajagtap) [inline README.md:1]:
```suggestion
# gorilla/context
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
 =======
```
Comment (apoorvajagtap) [inline README.md:2]:
```suggestion

```

**Review by apoorvajagtap — CHANGES_REQUESTED** [review]

**Comment by coreydaley** [comment]
@apoorvajagtap pull request updated, ready for a re-review

**Pull request merged**